src/main/java/com/revature/caliber/controllers/AssessmentController.java

```diff
@@ -49,6 +49,10 @@
         if(weekNumber != null) temp = as.findAssessmentsByBatchIdAndWeekNumber(id, weekNumber);
         else temp = as.findAssessmentsByBatchId(id);
         
+        for(Assessment ass : temp) {
```

Comment:
Is this intended to be in the codebase or was this just used for testing purposes?